tests_metricflow/snapshots/test_derived_metric_rendering.py/SqlQueryPlan/DuckDB/test_nested_filters__plan0_optimized.sql

```diff
@@ -8,9 +8,9 @@ FROM (
   FROM (
     -- Combine Aggregated Outputs
     SELECT
-      MAX(subq_48.average_booking_value) AS average_booking_value
-      , MAX(subq_61.bookings) AS bookings
-      , MAX(subq_69.booking_value) AS booking_value
+      MAX(subq_45.average_booking_value) AS average_booking_value
```

Comment:
The remaining snapshots should have the following pattern:

1. The standard snapshots should lose the duplicated where constraint subqueries that the original pushdown implementation added to the DataflowPlanBuilder. I added a comment in tests_metricflow/snapshots/test_predicate_pushdown_rendering.py/SqlQueryPlan/DuckDB/test_single_categorical_dimension_pushdown__plan0.sql to illustrate this.
2. The optimized snapshots will have a bunch of subquery ID updates due to the removal of the subqueries from the un-optimized plans. The should not include other changes.